src/bicep/mlz.bicep

```
@@ -252,6 +252,58 @@ module sharedServicesVirtualNetworkPeering './modules/spokeNetworkPeering.bicep'
   }
 }
 
+module hubPolicyAssignment './modules/policyAssignment.bicep' = {
+  name: 'policyAssignement'
```

nit: from the deployment output I couldn't tell these apart from the other assignments applied at the same time

```suggestion
  name: 'policyAssignment-${hubResourceGroupName}'
```